main.tf

```diff
@@ -118,7 +118,7 @@ resource "aws_cloudwatch_metric_alarm" "cache_memory" {
 
 module "dns" {
   source    = "git::https://github.com/cloudposse/terraform-aws-route53-cluster-hostname.git?ref=tags/0.2.1"
-  enabled   = "${var.enabled}"
+  enabled   = "${var.enabled == "true" && var.zone_id != "false"}"
```

Comment:
we usually use this pattern 
https://github.com/cloudposse/terraform-aws-rds-cluster/blob/master/main.tf#L124

but the one from @MoonMoon1919  is ok as well

Reply:
I think he did this because `zone_id` was previously set to `false` in a different PR. 

https://github.com/cloudposse/terraform-aws-elasticache-redis/pull/13/files#diff-c9ac8098c5ea9d3e6a9a596ff0c512a4R107

Agree we should change this to our normal pattern.

Reply:
Yeah, that's why I used this approach. Tonight I can remove the default value and submit a PR to address this. https://github.com/cloudposse/terraform-aws-elasticache-redis/issues/26